Gemfile

```diff
@@ -101,4 +101,6 @@ group :development do
   gem 'spork', git: 'https://github.com/sporkrb/spork', ref: '224df49' # '~> 1.0rc'
   gem 'spring'
   gem 'spring-commands-rspec'
+  gem 'ruby-debug-ide', '~> 0.7.3'
```

Comment:
We removed these in https://github.com/cloudfoundry/cloud_controller_ng/pull/3175 because

> We face an issue where, when bundling for the first time on a new env, the first attempt at installing ruby-debug-ide fails and bundling a second time magically works.

So might be worth checking in on that

Reply:
The thing is without this beeing in the gemfile VSCode refuses to work at all forcing everyone that uses that ide to temporarily add it to the gemfile anyway hmmm 

```
bundler: failed to load command: rdebug-ide (/Users/me/.gem/ruby/3.1.2/bin/rdebug-ide)
/Users/me/.gem/ruby/3.1.2/gems/bundler-2.4.10/lib/bundler/rubygems_integration.rb:308:in `block in replace_bin_path': can't find executable rdebug-ide for gem ruby-debug-ide. ruby-debug-ide is not currently included in the bundle, perhaps you meant to add it to your Gemfile? (Gem::Exception)
	from /Users/me/.gem/ruby/3.1.2/gems/bundler-2.4.10/lib/bundler/rubygems_integration.rb:336:in `block in replace_bin_path'
```

I havent had anny issue installing either ruby-debug-ide nor debase. So i would propose we try adding it to the dev dependencies again (i mean they are for real dev dependencies i would say) and see if theres still an issue. Maybe it was just a broken ruby-debug-ide version that wasnt properly installable.

Reply:
I found a way to make vscode use `rdbg` so the default debug gem from ruby 3 👍🏼